companion/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
-	"id": "your-module-name",
-	"name": "your-module-name",
-	"shortname": "module-shortname",
-	"description": "A short one line description of your module",
+	"id": "sennheiser-adn-mediactrl",
+	"name": "Sennheiser ADN MediaCtrl",
+	"shortname": "adn",
+	"description": "Make use of Sennheiser ADN MediaCtrl",
 	"version": "0.0.0",
 	"license": "MIT",
-	"repository": "git+https://github.com/bitfocus/companion-module-your-module-name.git",
-	"bugs": "https://github.com/bitfocus/companion-module-your-module-name/issues",
+	"repository": "git+https://github.com/bitfocus/companion-module-sennheiser-adn-mediactrl.git",
+	"bugs": "https://github.com/bitfocus/companion-module-sennheiser-adn-mediactrl/issues",
 	"maintainers": [
 		{
```

Comment:
Sorry it seems my own review didn't save!

This wants updating too!

But it didn't feel like my place to put something in, and I wasn't sure if stuff would break if there was no maintainer listed...

Reply:
this module currently looks like just an initial commit right now, so I don't think it matters really.
I only merged it so that it doesnt sit open